### DISCUSSION TITLE
Replace background `rect` with a `path` to save 6 bytes

### DISCUSSION THREAD

**Discussion by kuubeu** [edent/SuperTinyIcons#730] [Ideas]
Since `rx="15%"` is no longer used for backgrounds, maybe we should stop using rects altogether? It would save 6 bytes on each icon which uses background rects.

Here's how they compare side by side:
```svg
<rect fill="tan" width="512" height="512"/>
<path fill="tan" d="M0 0H512V512H0"/>
```

**Comment by edent** [comment]
Sounds good. Try it and see!

**Comment by kuubeu** [comment]
Closed by #758
